src/core/ddsc/tests/qos_provider.c

```diff
@@ -391,12 +391,13 @@ static uint32_t b64_encode (const unsigned char *text, const uint32_t sz, unsign
   for (size_t i = 0, j = 0; i < buff_len && j < sz; i+=4, j+=3)
   {
     unsigned char chunk[4] = {0x00, 0x00, 0x00, 0x00};
-    chunk[0] = base64_etable[(text[j] >> 0x02U)];
```

Comment:
Ah, this one. I remember it from C++ 🙂

Can you split the commit so that this becomes a separate one?